subjects/simple_hash/README.md

```diff
@@ -35,16 +35,16 @@ fn main() {
 
     println!(
         "Does the HashMap contains the name Roman? => {}",
-        contain(&hash, "Roman")
+        contain(hash.clone(), "Roman")
```

Comment:
There is no need to clone the hash map. Just borrowing it is enough. I'm afraid this is going to enforce a sub-optimal way of writing Rust. There is no need to take ownership of a value if you only want to read the its content. But this is more with @LEEDASILVA who made the exercise

Reply:
hey i did not do this exercise, maybe @MSilva95?

Reply:
Ah ok, my bad @LEEDASILVA . I went and I change it myself. The solution, test and subject are all up to date now.